eap-xp4-jdk17/runtime-image/image.yaml

```diff
@@ -0,0 +1,90 @@
+schema_version: 1
+
+name: "jboss-eap-7/eap-xp4-openjdk11-runtime-openshift-rhel8"
+description: "Red Hat JBoss Enterprise Application Platform XP 4.0 OpenShift runtime image with OpenJDK 11"
+version: "4.0"
+from: "registry.redhat.io/ubi8/ubi:latest"
+labels:
+    - name: "com.redhat.component"
+      value: "jboss-eap-xp4-openjdk11-runtime-openshift-rhel8-container"
+    - name: "io.k8s.description"
+      value: "Base runtime image to run EAP XP server and application"
+    - name: "io.k8s.display-name"
+      value: "JBoss EAP XP runtime image"
+    - name: "io.openshift.expose-services"
+      value: "8080:http"
+    - name: "io.openshift.tags"
+      value: "javaee,eap,eap7"
+    - name: "maintainer"
+      value: "Red Hat"
+envs:
+    - name: "LAUNCH_JBOSS_IN_BACKGROUND"
+      value: "true"
+    - name: HTTPS_ENABLE_HTTP2
+      value: "true"
+    - name: SCRIPT_DEBUG
+      description: If set to true, ensures that the bash scripts are executed with the -x option, printing the commands and their arguments as they are executed.
+      example: "true"
+    - name: JBOSS_HOME
+      value: "/opt/eap"
+    # env var set by modules in the builder image context, duplicating them here.
+    - name: JBOSS_MODULES_SYSTEM_PKGS
+      value: "jdk.nashorn.api"
+    - name: DEFAULT_ADMIN_USERNAME
+      value: "eapadmin"
+    - name: MICROPROFILE_CONFIG_DIR_ORDINAL
+      value: "500"
+ports:
+    - value: 8443
+    - value: 8080
+    - value: 8787
+      expose: false
+modules:
+      repositories:
```

Comment:
We are missing the jdk module like : https://github.com/jboss-container-images/jboss-eap-7-openshift-image/pull/506/files#diff-9c0299c8822804204d2aeed9e6d492309692485079e76eed76be5d6786e8be14R56